static/app/components/feedback/useMutateActivity.tsx

```diff
@@ -41,13 +70,11 @@ export default function useMutateFeedbackActivity({organization, group}: Props)
         {text: note?.text, mentions: note?.mentions},
       ]);
     },
-    onSettled: (_resp, _error, _var, _context) => {
-      invalidateCached([group.id]);
-    },
+    onSettled: onSettled ?? undefined,
     cacheTime: 0,
   });
 
-  const updateComment = useCallback(
+  const handleUpdate: UpdateCommentCallback = useCallback(
```

Comment:
defining these types twice. add it as the generic in useCallback remove the argument types below

```suggestion
  const handleUpdate = useCallback<UpdateCommentCallback>(
    (note, noteId, activity, options) => {
```